### ISSUE TITLE
Tiles don't update when using Collection of Images tileset

### ISSUE THREAD

**Issue by EdenIndustries** [bjorn/tiled#990]
If you edit a tile that is included in a Collection of Images tileset using an external image editing program, it won't be updated within Tiled. However, if you edit a tile that is part of a tileset Based on a Tileset Image, it will be updated within Tiled.

**Comment by bjorn** [comment]
Right, that functionality is currently only hooked up to full tileset images. It should really be extended to the invidual tiles as well, but it will take some effort to keep the code clean.
